packages/suite/src/components/suite/modals/PassphraseSource/index.tsx

```diff
@@ -35,7 +32,6 @@ const PassphraseSource = ({ device, getDiscoveryAuthConfirmationStatus, ...rest
                     />
                 }
                 data-test="@modal/confirm-empty-hidden-wallet"
-                {...rest}
```

Comment:
Same here, I'm not sure if it's not used.

Reply:
65f1c90